core/zoe/zoe.chainmail

```diff
@@ -0,0 +1,163 @@
+/**
+ * This is returned by a call to `makeInstance` on Zoe.
+ */
+struct UserFacetAndGoverningContract ( ) {
+  zoeInstance :ZoeUserFacet;
+  governingContract :CustomGoverningContract;
+}
+
+/**
+ * This is returned by a call to `escrow` on the zoeInstance userFacet.
+ * An escrowReceipt is an ERTP payment which acts as proof of escrow
+ * and is normally passed in to a governingContract method. A
+ * claimPayoff ERTP payment is what can be unwrapped to claim the
+ * payoff of involvement with a contract. 
+ */
+struct UserFacetAndGoverningContract ( ) {
```

Comment:
wrong name, I think you mean EscrowReceiptAndClaimPayoff.

When we teach the chainmail parser to do something with its parse tree, we'll catch this redundant declaration.

Reply:
Yup, good catch